example-app/src/js/biometric-tester.js

```
@@ -0,0 +1,397 @@
+    if (!code) {
+      showResult(elements.debugOutput, '❌ Please enter some code to execute', 'error');
+      return;
+    }
+
+    try {
+      addConsoleLog('INFO', [`Executing debug code: ${code}`]);
+
+      // Simple eval for testing (be careful with this in production!)
+      const result = await eval(`(async () => { ${code} })()`);
+
+      addConsoleLog('SUCCESS', ['✅ Debug code executed successfully:', result]);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)